ios build error:
```
Release-iphoneos/opencv2.build/Objects-normal/arm64/CalibExt.o -embed-bitcode-marker
2026-06-10T08:00:45.0846120Z <module-includes>:1:9: note: in file included from <module-includes>:1:
2026-06-10T08:00:45.0846670Z #import "Headers/opencv2.h"
2026-06-10T08:00:45.0846920Z         ^
2026-06-10T08:00:45.0848070Z /Users/opencv-cn/GHA-OCV-5/_work/opencv/opencv/ios/build/build-arm64-iphoneos/lib/Release/opencv2.framework/Headers/opencv2.h:111:9: note: in file included from /Users/opencv-cn/GHA-OCV-5/_work/opencv/opencv/ios/build/build-arm64-iphoneos/lib/Release/opencv2.framework/Headers/opencv2.h:111:
2026-06-10T08:00:45.0849800Z #import <opencv2/Volume.h>
2026-06-10T08:00:45.0850050Z         ^
2026-06-10T08:00:45.0850690Z /Users/opencv-cn/GHA-OCV-5/_work/opencv/opencv/ios/build/build-arm64-iphoneos/lib/Release/opencv2.framework/Headers/Volume.h:63:32: error: expected a type
2026-06-10T08:00:45.0852060Z - (instancetype)initWithVtype:(VolumeType)vtype settings:(VolumeSettings*)settings;
2026-06-10T08:00:45.0852520Z                                ^
2026-06-10T08:00:45.0853020Z <module-includes>:1:9: note: in file included from <module-includes>:1:
2026-06-10T08:00:45.0853420Z #import "Headers/opencv2.h"
2026-06-10T08:00:45.0853650Z         ^
2026-06-10T08:00:45.0854810Z /Users/opencv-cn/GHA-OCV-5/_work/opencv/opencv/ios/build/build-arm64-iphoneos/lib/Release/opencv2.framework/Headers/opencv2.h:111:9: note: in file included from /Users/opencv-cn/GHA-OCV-5/_work/opencv/opencv/ios/build/build-arm64-iphoneos/lib/Release/opencv2.framework/Headers/opencv2.h:111:
2026-06-10T08:00:45.0856070Z #import <opencv2/Volume.h>
2026-06-10T08:00:45.0856290Z         ^
2026-06-10T08:00:45.0856930Z /Users/opencv-cn/GHA-OCV-5/_work/opencv/opencv/ios/build/build-arm64-iphoneos/lib/Release/opencv2.framework/Headers/Volume.h:69:32: error: expected a type
2026-06-10T08:00:45.0857690Z - (instancetype)initWithVtype:(VolumeType)vtype;
2026-06-10T08:00:45.0858010Z                                ^
2026-06-10T08:00:45.0858360Z <module-includes>:1:9: note: in file included from <module-includes>:1:
2026-06-10T08:00:45.0858760Z #import "Headers/opencv2.h"
2026-06-10T08:00:45.0858990Z         ^
2026-06-10T08:00:45.0860150Z /Users/opencv-cn/GHA-OCV-5/_work/opencv/opencv/ios/build/build-arm64-iphoneos/lib/Release/opencv2.framework/Headers/opencv2.h:112:9: note: in file included from /Users/opencv-cn/GHA-OCV-5/_work/opencv/opencv/ios/build/build-arm64-iphoneos/lib/Release/opencv2.framework/Headers/opencv2.h:112:
2026-06-10T08:00:45.0861560Z #import <opencv2/VolumeSettings.h>
2026-06-10T08:00:45.0861820Z         ^
2026-06-10T08:00:45.0862490Z /Users/opencv-cn/GHA-OCV-5/_work/opencv/opencv/ios/build/build-arm64-iphoneos/lib/Release/opencv2.framework/Headers/VolumeSettings.h:53:37: error: expected a type
2026-06-10T08:00:45.0863320Z - (instancetype)initWithVolumeType:(VolumeType)volumeType;
2026-06-10T08:00:45.0863670Z                                     ^
2026-06-10T08:00:45.0864010Z <unknown>:0: error: could not build Objective-C module 'opencv2'
2026-06-10T08:00:45.0864310Z 
```

--- a/modules/objc/generator/gen_objc.py
+++ b/modules/objc/generator/gen_objc.py
@@ -847,13 +847,14 @@ class ObjectiveCWrapperGenerator(object):
             objc_type = enumType.rsplit(".", 1)[-1]
             if objc_type in enum_ignore_list:
                 return
-            if not constinfo.classname:
+            original_objc_type = objc_type
+            if not constinfo.classname or constinfo.classname == objc_type:
                 module_fix = enum_fix.get(self.Module, {})
                 objc_type = module_fix.get(objc_type, objc_type)
                 objc_type = enum_fix.get(objc_type, objc_type)
             elif constinfo.classname in enum_fix:
                 objc_type = enum_fix[constinfo.classname].get(objc_type, objc_type)
-            import_module = constinfo.classname if constinfo.classname and constinfo.classname != objc_type else self.Module
+            import_module = constinfo.classname if constinfo.classname and constinfo.classname != original_objc_type else self.Module
             type_dict[ctype] = { "cast_from" : "int",
                                  "cast_to" : get_cname(enumType),
                                  "objc_type" : objc_type,

--- a/modules/ptcloud/misc/objc/gen_dict.json
+++ b/modules/ptcloud/misc/objc/gen_dict.json
@@ -1,7 +1,7 @@
 {
   "enum_fix": {
     "Ptcloud": {
-      "VolumeType": "PtcloudVolumeType"
+      "VolumeType": "CV_VolumeType"
     }
   }
 }

